app/assets/stylesheets/bl-overrides.scss

```diff
@@ -90,7 +90,7 @@ body.public-facing.bl-public-facing {
   }
 
   .pt-20 {
-    padding-top: 20px;
+    padding-top: 60px;
```

Comment:
the name of this class is pt-20, meaning padding-top: 20px. If you change this to 60px, it will change everywhere it is used throughout the app which will likely break the layout in other places.

just looking at how the rest of this file is set up, it would make more sense to add a pt-60 class and then use that instead.

Also, make sure that 60px is the right amount of top padding for each breakpoint. you may need to add media queries for smaller screen sizes.

Reply:
Done